python/mathsys/dev/parser/dataclasses.py

```
@@ -0,0 +1,151 @@
+#> START -> CLASS
+@dataclass
+class Start:
+    statements: list[Level1]
+
+
+#^
+#^  1ºLEVEL
+#^
+
+#> 1ºLEVEL -> NAMESPACE
+class Level1: pass
```

Ruff (0.14.6)

Multiple statements on one line (colon)

(E701)